### PR TITLE
Slight modifications to wording on home page

### PR DESCRIPTION
**Pull request by tyrionmcmaster**
Improving accuracy and wording of information on home page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,8 +68,8 @@ <h3>Monero is untraceable.</h3>
                             <img src="/img/fungible-monero.png" alt="Monero gold" class="main-icon">
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
-                            <h3>Monero is fungible.</h3>
-                            <p> Monero is fungible because it is private by default. In its current state, it is extremely unlikely that Monero will ever be blacklisted by exchanges or vendors due to its association in previous transactions.</p>
+                            <h3>Monero is fungible</h3>
+                            <p> Monero is fungible <a href="https://getmonero.org/resources/moneropedia/fungibility.html">because it is private by default. Units of Monero cannot be blacklisted by vendors or exchanges due to their association in previous transactions.</p>
```
Comment (MaxXor) [inline index.html]: This link got no closing tag and should be around the word `fungible`.

File: index.html [hunk 2]
```diff
@@ -49,14 +49,14 @@ <h3>Monero is secure.</h3>
                             <img src="/img/private-monero.png" alt="No surveillance" class="main-icon">
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
-                            <h3>Monero is private.</h3>
-                            <p>Monero uses ring signatures and ring confidential transactions to obfuscate the amounts, origins, and destinations of all transactions. Monero provides all the benefits of a decentralized cryptocurrency, without any of the typical privacy concessions. </p>
+                            <h3>Monero is private</h3>
+                            <p>Monero uses ring signatures, ring confidential transactions, and stealth addresses to obfuscate the origins, amounts and destinations of all transactions. Monero provides all the benefits of a decentralized cryptocurrency, without any of the typical privacy concessions. </p>
```
Comment (MaxXor) [inline index.html]: `, and` -> ` and`

File: index.html [hunk 3]
```diff
@@ -68,8 +68,8 @@ <h3>Monero is untraceable.</h3>
                             <img src="/img/fungible-monero.png" alt="Monero gold" class="main-icon">
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
-                            <h3>Monero is fungible.</h3>
-                            <p> Monero is fungible because it is private by default. In its current state, it is extremely unlikely that Monero will ever be blacklisted by exchanges or vendors due to its association in previous transactions.</p>
+                            <h3>Monero is fungible</h3>
+                            <p> Monero is <a href="https://getmonero.org/resources/moneropedia/fungibility.html"> fungible </a>because it is private by default. Units of Monero cannot be blacklisted by vendors or exchanges due to their association in previous transactions.</p>
```
Comment (MaxXor) [inline index.html]: You can shorten the URL to `/resources/moneropedia/fungibility.html` so it works also in local development environment. Also usually the spaces are outside the `<a>`-tag so you don't introduce double spaces..

**Comment by SamsungGalaxyPlayer** [comment]
@MaxXor @tyrionmcmaster I'm highly in support of retaining the Oxford comma.

File: index.html [hunk 4]
```diff
@@ -49,14 +49,14 @@ <h3>Monero is secure.</h3>
                             <img src="/img/private-monero.png" alt="No surveillance" class="main-icon">
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
-                            <h3>Monero is private.</h3>
-                            <p>Monero uses ring signatures and ring confidential transactions to obfuscate the amounts, origins, and destinations of all transactions. Monero provides all the benefits of a decentralized cryptocurrency, without any of the typical privacy concessions. </p>
+                            <h3>Monero is private</h3>
+                            <p>Monero uses ring signatures, ring confidential transactions and stealth addresses to obfuscate the origins, amounts and destinations of all transactions. Monero provides all the benefits of a decentralized cryptocurrency, without any of the typical privacy concessions. </p>
```
Comment (luigi1111) [inline index.html]: I don't know if we have a style guide that includes reference to the Oxford comma, but as the current dictator of this repo I demand it be used (or shown the guide). :)

I'm also not sure of the language around "ring signatures, ring confidential transactions..." as the former is subsumed by the latter.

**Pull request merged**